### PR TITLE
 clean up code in module autoconfigure, controller and dependency.

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* Lombok replacement
* Add empty line to separate code blocks
* Allow one line with 120 chars to eliminate wrapper.

Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

File: src/main/java/com/microsoft/azure/springcloudplayground/controller/AbstractPlaygroundController.java [hunk 1]
```diff
@@ -50,40 +51,37 @@ public void invalidProjectRequest(HttpServletResponse response,
...
 
         // Google analytics support
-        model.put("trackingCode",
-                metadata.getConfiguration().getEnv().getGoogleAnalyticsTrackingCode());
-
-    }
-
-    public Function<String, String> getLinkTo() {
-        return this.linkTo;
+        model.put("trackingCode", metadata.getConfiguration().getEnv().getGoogleAnalyticsTrackingCode());
```
Comment (yaweiw) [inline src/main/java/com/microsoft/azure/springcloudplayground/controller/AbstractPlaygroundController.java]: let's remove this?

File: src/main/java/com/microsoft/azure/springcloudplayground/dependency/Dependency.java [hunk 2]
```diff
@@ -90,11 +121,10 @@
 
     private List<String> keywords = new ArrayList<>();
 
+    @Getter
+    @Setter
     private List<Link> links = new ArrayList<>();
 
-    public Dependency() {
```
Comment (yaweiw) [inline src/main/java/com/microsoft/azure/springcloudplayground/dependency/Dependency.java]: Is this constructor needed?

File: src/main/java/com/microsoft/azure/springcloudplayground/controller/AbstractPlaygroundController.java (hunk 1 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/controller/AbstractPlaygroundController.java]: OK, that will the next step for cleanup. For simply, this PR is avoiding touch the details of code implementation.

File: src/main/java/com/microsoft/azure/springcloudplayground/dependency/Dependency.java (hunk 2 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/dependency/Dependency.java]: Needed, `@NoArgsConstructor` will cover this.

**Pull request merged**